lib/ae_mdw/activities.ex

```diff
@@ -657,13 +663,10 @@ defmodule AeMdw.Activities do
   end
 
   defp render_payload(state, _account_pk, _height, txi, {:claim, local_idx}) do
-    Model.tx(time: micro_time) = State.fetch!(state, Model.Tx, txi)
-
     {claim_aetx, :name_claim_tx, tx_hash, tx_type, _block_hash} =
       DbUtil.read_node_tx_details(state, {txi, local_idx})
 
     payload = %{
-      micro_time: micro_time,
```

Comment:
Better keep it to avoid potential breaking changes (added more than 6 months ago)

Reply:
Good call, done